Makefile

```diff
@@ -348,9 +348,9 @@ binary-stacker: $(STACKER)
 		--substitute RELEASE_TAG=$(RELEASE_TAG) \
 		--substitute REPO_NAME=zot-$(OS)-$(ARCH)
 
-.PHONY: image
-image:
-	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -f build/Dockerfile -t zot:latest .
+.PHONY: docker-image
+docker-image:
+	${CONTAINER_RUNTIME} buildx build --platform $(OS)/$(ARCH) -f build/Dockerfile .
```

Comment:
Does this work with podman? Locally (on my side) this target fails.
Should we use docker directly?

Reply:
$ make CONTAINER_RUNTIME=podman docker-image

Reply:
https://github.com/rchincha/zot/actions/runs/6396870937/job/17363595069

podman works